sql/core/src/test/scala/org/apache/spark/sql/MathFunctionsSuite.scala

```diff
@@ -244,7 +244,7 @@ class MathFunctionsSuite extends QueryTest with SharedSparkSession {
     }
   }
 
-  test("SPARK-36229 inconsistently behaviour where returned value is above the 64 char threshold") {
+  test("SPARK-36229: inconsistent behaviour where returned value is above the 64 char threshold") {
```

Comment:
Please revert irrelevant style changes from this PR. It makes backporting your PR difficult.

Reply:
Done